src/frontend/context/fagsak/FagsakContext.tsx

```diff
@@ -138,16 +119,15 @@ export const FagsakProvider = (props: PropsWithChildren) => {
     };
 
     const oppdaterTilbakekrevingsbehandlingerPåFagsak = () => {
-        const fagsakId = hentDataFraRessurs(minimalFagsakRessurs)?.id;
+        const fagsakId = fagsak?.id;
```

Comment:
Her vet man vel at fagsak er definert 

```suggestion
        const fagsakId = fagsak.id;
```

Du kan og vurdere å skrive om til noe ala dette:

```
const oppdaterTilbakekrevingsbehandlingerPåFagsak = () => {
        hentTilbakekrevingsbehandlinger(fagsak.id).then(tilbakekrevingsbehandlingerRessurs =>
            settTilbakekrevingsbehandlinger(tilbakekrevingsbehandlingerRessurs)
        );
    };
```